autosklearn/automl.py

```diff
@@ -626,7 +628,7 @@ def fit_ensemble(self, y, task=None, metric=None, precision='32',
 
     def _get_ensemble_process(self, time_left_for_ensembles,
                               task=None, metric=None, precision=None,
-                              dataset_name=None, max_iterations=-1,
+                              dataset_name=None, max_iterations=None,
```

Comment:
I changed this, because I didn't understand what would be the behavior for max_iterations=-1. As far as I understood, the value that is used to deactivate the max_iterations limit is None.

Reply:
Yes, I agree.